src/instructlab/configuration.py

```diff
@@ -540,6 +540,32 @@ class _train(BaseModel):
     )
 
 
+class _metadata(BaseModel):
+    # model configuration
+    model_config = ConfigDict(
+        extra="ignore",
+        protected_namespaces=(),
+        use_enum_values=True,  # populate models with the value property of enums, rather than the raw enum.
+    )
+    cpu_info: str | None = Field(
+        default=None,
+        description="Manufacturer, Family, and SKU of the system CPU, ex: Apple M3 Max",
+    )
+    gpu_manufacturer: str | None = Field(
+        default=None, description="Manufacturer of the system GPU, ex: Nvidia"
```

Comment:
Consider using enum if we want to keep gpu_manufacturer to a fixed set of possible values 

```
from enum import Enum

class GPUManufacturer(str, Enum):
    NVIDIA = "Nvidia"
    AMD = "AMD"
    INTEL = "Intel"

class SystemMetadata(BaseModel):
    gpu_manufacturer: GPUManufacturer | None = Field(
        default=None, description="Manufacturer of the system GPU, ex: Nvidia"
    )

```

Reply:
I'd like to avoid using enums because they introduce issues with click.